functional-samples/tutorial.tabs-manager/popup.js

```diff
@@ -14,8 +14,8 @@
 
 const tabs = await chrome.tabs.query({
   url: [
-    'https://developer.chrome.com/docs/webstore/*',
-    'https://developer.chrome.com/docs/extensions/*'
+    'https://developer.chrome.com/docs/extensions',
```

Comment:
Could you explain why you made this change? My initial reaction is that we shouldn't go ahead with it.

Reply:
Hi @oliverdunk ,

Both https://developer.chrome.com/docs/extensions/* and https://developer.chrome.com/docs/webstore/* lead to 404 Not Found page, so I thought it would be good to have some valid URLs.

![image](https://github.com/user-attachments/assets/e4663cb9-7a2a-4b3e-bd35-0535fe3d0c06)

Reply:
Sorry, I misunderstood. You are right. Those asterisks were for the wildcard. Reverting this change.